examples/abort/abort-all-requests.ts

```diff
@@ -0,0 +1,55 @@
+import ollama from 'ollama'
+
+// Set a timeout to abort all requests after 1 second
+setTimeout(() => {
+  console.log('\nAborting all requests...\n')
+  ollama.abort()
+}, 1000) // 1000 milliseconds = 1 second
```

Comment:
```suggestion
// Set a timeout to abort all requests after 5 seconds
setTimeout(() => {
  console.log('\nAborting all requests...\n')
  ollama.abort()
}, 5000) // 5000 milliseconds = 5 seconds
```

This was 1 second previously, but I'd like to give it a bit more time to load the model into memory.

Reply:
That's a good idea!